### DISCUSSION TITLE
Is GitHub working with Greenhouse?

### DISCUSSION THREAD

**Discussion by kemjim** [community/community#22225] [New to GitHub]
<p>Hello,<br>
I received an email from ‘GitHub Talent Acquisition’ but the email address is “<a href="mailto:no-reply@greenhouse.io">no-reply@greenhouse.io</a>”. The talent acquisition team is asking for a consent to access my personal GitHub information.<br>
While I trust GitHub, I have not heard “Greenhouse IO”. Since the email is coming from Greenhouse, I would like to confirm its legitimacy.<br>
Can anyone confirm that GitHub is working with Greenhouse for recruiting?</p>
<p>Thanks,<br>
Kemal</p>

**Comment by timoyates** [comment]
<p>Hi <a class="mention" href="/alaeddinoglu">@alaeddinoglu</a>, that sounds legitimate, but have shared a link to this post with our TA team so that they can confirm.</p>

**Comment by kemjim** [comment]
<p>Thank you for replying so fast!</p>

**Comment by timoyates** [comment]
<p><a class="mention" href="/alaeddinoglu">@alaeddinoglu</a> You’re welcome :relaxed:</p>
<p>Here’s what I got back: <em>This is a legitimate email sent from our Greenhouse system. In order to be GDPR compliant we are now required to give all past, current, and future candidates the opportunity to consent to us keeping or deleting their data within Greenhouse. If you Give consent we will keep your data on file for another 365 days. If you Deny consent, we will delete your data from Greenhouse completely. Please let me know if you have any additional questions or concerns.</em></p>
<p>My colleague also wrote that she’d posted something similar here directly on the post, but I’m not seeing it yet, so I decided to go ahead and share the response for now so you’re not waiting around in the dark.</p>

**Comment by abentley13** [comment, marked as answer]
<p>Hello all!  This is a legitimate email sent from our Greenhouse system. In order to be GDPR compliant we are now required to give all past, current, and future candidates the opportunity to consent to us keeping or deleting their data within Greenhouse. If you Give consent we will keep your data on file for another 365 days. If you Deny consent, we will delete your data from Greenhouse completely. Please let me know if you have any additional questions or concerns.</p>

**Comment by jks2185** [comment]
<p>If I receive this email, is that for a current position or future ones? I applied for a position a month ago and then just received this email, but haven’t received anything stating I havent been selected yet.</p>

**Comment by abentley13** [comment]
<p>Hi <a class="mention" href="/jks2185">@jks2185</a> - This email is separate from whether or not you have been selected. To be compliant, we are required to send the GDPR Consent email to all who have applied to GitHub in the past and recently. Going forward compliance will be captured via a Checkbox on all applications. If you wish to still be considered for the role you applied to, please consent to us storing your data or we will not be able to move forward with your application.</p>

**Comment by jks2185** [comment]
<p>Thanks, I consented. Is there any way to check on the progress of an application by the way?</p>
